packages/patternfly-4/react-docs/src/components/example/example.js

```diff
@@ -102,13 +102,13 @@ const Example = ({
           />
         </React.Fragment>
       ) : (
-        <React.Fragment>
-          <div className={css(className, styles.example)} {...props}>
-            {children}
-          </div>
-          <LiveDemo raw={raw.trim()} path={examplePath} live={false} />
-        </React.Fragment>
-      )}
+          <React.Fragment>
+            <div className={css(className, styles.example)} {...props}>
+              {children}
+            </div>
+            <LiveDemo raw={raw.trim()} path={examplePath} live={false} />
+          </React.Fragment>
+        )}
```

Comment:
Any reason for the extra indentation here?

Reply:
Nope! My IDE planted them there from file save auto-formatting, ironically enough I didn't have the "Format on Save" setting set, switching it on still doesn't allow me to undo it. Weird. Can remove though.

It brings up a good point though, in the other PF repos we use prettier, but in PF4 I guess not?

_seanforyou23 shruggs_